lib/bemxjst/entity.js

```diff
@@ -150,32 +102,17 @@ Entity.prototype.run = function run(context) {
   return this.defaultBody(context);
 };
 
-Entity.prototype.defaultBody = function defaultBody(context) {
-  var tag = context.ctx.tag;
-  if (tag === undefined)
-    tag = this.tag.exec(context);
-
-  var js;
-  if (context.ctx.js !== false)
-    js = this.js.exec(context);
-
-  var bem = this.bem.exec(context);
-  var cls = this.cls.exec(context);
-  var mix = this.mix.exec(context);
-  var attrs = this.attrs.exec(context);
-  var content = this.content.exec(context);
-
-  // Default content
-  if (this.content.count === 0 && content === undefined)
-    content = context.ctx.content;
-
-  return this.bemhtml.render(context,
-                             this,
-                             tag,
-                             js,
-                             bem,
-                             cls,
-                             mix,
-                             attrs,
-                             content);
+Entity.prototype.setDefaults = function setDefaults() {
+  // Default .content() template for applyNext()
+  if (this.content.count !== 0)
+    this.content.push(new Template([], contentMode));
+
+  // .def() default
+  if (this.def.count !== 0) {
+    this.canFlush = this.options.flush || false;
```

Comment:
This is still set here, but the property is declared only on a child class. What do you think about moving the property back to this file?